ci-operator/step-registry/rosa/sts/operator-roles/delete/rosa-sts-operator-roles-delete-commands.sh

```
@@ -45,7 +70,14 @@ if [[ -e "${OPERATOR_ROLES_PREFIX_FILE}" ]]; then
   OPERATOR_ROLES_PREFIX=$(cat "${OPERATOR_ROLES_PREFIX_FILE}")
 
   echo "Start deleting the byo operator roles with the prefix ${OPERATOR_ROLES_PREFIX}..."
-  rosa delete operator-roles -y --mode auto --prefix ${OPERATOR_ROLES_PREFIX}
+  set +e
+  retry_with_backoff 3 rosa delete operator-roles -y --mode auto --prefix ${OPERATOR_ROLES_PREFIX}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)